library/aws/aws.ts

```diff
@@ -124,6 +124,44 @@ export function createS3Bucket(
   return AR.createS3Bucket(tfgen, name, params);
 }
 
+// CreateS3Bucket for newer versions of terraform
+export function createS3BucketV2(
```

Comment:
Bit too many positional args. Can you move them into sub-struct?

Reply:
done :)